src/plugins/data/server/saved_objects/query.ts

```diff
@@ -11,7 +11,8 @@ import { SavedObjectsType } from 'kibana/server';
 export const querySavedObjectType: SavedObjectsType = {
   name: 'query',
   hidden: false,
-  namespaceType: 'single',
+  namespaceType: 'multiple-isolated',
+  convertToMultiNamespaceTypeVersion: '8.0.0',
```

Comment:
I think you accidentally included all of the other changes that are in the other PR, too.

This PR should only need these changes.

Reply:
Yeah, I did this so I don't forget to merge the other PR first before merging this one